openmc/mgxs/mgxs.py

```diff
@@ -2777,19 +2990,43 @@ def __init__(self, domain=None, domain_type=None,
 
 
 class NuScatterXS(MGXS):
-    """A nu-scatter multi-group cross section.
+    r"""A scattering neutron production multi-group cross section.
+
+    The neutron production from scattering is defined as the average number of
+    neutrons produced from all neutron-producing reactions except for fission.
 
     This class can be used for both OpenMC input generation and tally data
     post-processing to compute spatially-homogenized and energy-integrated
-    multi-group cross sections for multi-group neutronics calculations.
+    multi-group cross sections for multi-group neutronics calculations. At a
+    minimum, one needs to set the :attr:`NuScatterXS.energy_groups` and
+    :attr:`NuScatterXS.domain` properties. Tallies for the flux and appropriate
+    reaction rates over the specified domain are generated automatically via the
+    :attr:`NuScatterXS.tallies` property, which can then be appended to a
+    :class:`openmc.Tallies` instance.
+
+    For post-processing, the :meth:`MGXS.load_from_statepoint` will pull in the
+    necessary data to compute multi-group cross sections from a
+    :class:`openmc.StatePoint` instance. The derived multi-group cross section
+    can then be obtained from the :attr:`NuScatterXS.xs_tally` property.
+
+    For a spatial domain :math:`D` and energy group :math:`[E_g,E_{g-1}]`, the
+    scattering neutron production cross section is calculated as:
+
+    .. math::
+
+       \frac{\int_{r \in D} dr \int_{4\pi} d\Omega \int_{E_g}^{E_{g-1}} dE \;
```

Comment:
What do you think about using "V" instead of "D" for spatial domain? It's not a big deal, just seems like it might be more intuitive in the math, though I can see the mapping from "domain" to "D".